example/mixin/alerts.yaml

```diff
@@ -61,3 +61,14 @@ groups:
     for: 5m
     labels:
       severity: warning
+  - alert: PrometheusOperatorRejectedResources
+    annotations:
+      description: Prometheus operator in {{ $labels.namespace }} namespace rejected
+        {{ printf "%0.0f" $value }} {{ $labels.controller }}/{{ $labels.resource }}
+        resources.
+      summary: Resources rejected
```

Comment:
nit: should we mention that this is about prometheus-operator resources in summary?

Reply:
yes sounds good to me